### PR TITLE
feature/SKOOP-165

### PR DESCRIPTION
**Pull request by LebedkoDmitry**
After successful login a user is redirected to a path where she / he came from.
`postLogoutRedirectUri` is set to redirect a user to a root path after successful logout.

@georgwittberger @svetivanova could you please take a look?

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -231,7 +231,9 @@ export function initAuthentication(moduleConfig: OAuthModuleConfig, oauthService
       redirectUri: environment.authentication.redirectUri ? environment.authentication.redirectUri : window.location.origin + '/',
```
Comment (georgwittberger) [inline src/app/app.module.ts]: Now that we set the `redirectUri` in each environment configuration we should remove the fallback here. The URI is simply a mandatory value in the environment settings now.

File: src/app/app.module.ts [hunk 2]
```diff
@@ -231,7 +231,9 @@ export function initAuthentication(moduleConfig: OAuthModuleConfig, oauthService
...
       silentRefreshRedirectUri: environment.authentication.silentRefreshRedirectUri,
       oidc: true,
-      requireHttps: environment.authentication.requireHttps
+      requireHttps: environment.authentication.requireHttps,
+      postLogoutRedirectUri: environment.authentication.postLogoutRedirectUri ?
```
Comment (georgwittberger) [inline src/app/app.module.ts]: For consistency I'd recommend to set the post-logout URI also in the environment config and remove the fallback here.

File: src/app/app.module.ts (hunk 1 above)
Comment (LebedkoDmitry) [reply src/app/app.module.ts]: Thank you for the suggestion 👍 

I have set `redirectUri` to `window.location.href` directly in the app module and removed the fallback.

File: src/app/app.module.ts (hunk 2 above)
Comment (LebedkoDmitry) [reply src/app/app.module.ts]: Thank you for the idea 👍 

I have set `postLogoutRedirectUri` to `window.location.origin + '/'` directly in the app module and removed the fallback.

**Pull request merged**